CMakeLists.txt

```diff
@@ -13,7 +13,8 @@
 # and the subproject (i.e: CbmRoot, Pandaroot,... etc).
 
 # Check if cmake has the required version
-CMAKE_MINIMUM_REQUIRED(VERSION 3.11.0 FATAL_ERROR)
+CMAKE_MINIMUM_REQUIRED(VERSION 3.12.0 FATAL_ERROR)
+set(BUILD_SHARED_LIBS ON)
```

Comment:
While this solves the issue, [this option has more implications](https://cmake.org/cmake/help/latest/variable/BUILD_SHARED_LIBS.html) which I would prefer to avoid - better set it as a `-D` command line option when configuring FairRoot. Also, it has been [acknowledged as a bug by upstream](https://github.com/boostorg/boost_install/issues/5) and fixed for next release.

Reply:
Ok.